Package.swift

```diff
@@ -61,7 +61,7 @@ let package = Package(
         .package(url: "https://github.com/SwiftDocOrg/GraphViz.git", exact: "0.2.0"),
         .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", exact: "2.10.1"),
         .package(url: "https://github.com/SwiftGen/SwiftGen", exact: "6.5.1"),
-        .package(url: "https://github.com/tuist/XcodeProj.git", exact: "8.11.0"),
+        .package(url: "https://github.com/tuist/XcodeProj.git", branch: "main"),
```

Comment:
A new release has been released and now you can link to it 🎉:
```swift
.package(url: "https://github.com/tuist/XcodeProj.git", exact: "8.13.0"),
```

Reply:
thx. done.